src/plugins/visualizations/public/visualize_app/components/visualize_top_nav.tsx

```diff
@@ -231,6 +269,19 @@ const TopNav = ({
       showSaveQuery={Boolean(services.visualizeCapabilities.saveQuery)}
       showSearchBar
       useDefaultBehaviors
+      badges={
+        !hideTryInLensBadge && displayEditInLensItem
+          ? [
+              {
+                badgeText: i18n.translate('visualizations.tonNavMenu.tryItBadgeText', {
+                  defaultMessage: 'Try it',
+                }),
+                color: 'accent',
+              },
+            ]
+          : []
+      }
```

Comment:
Very good point Michael!! I moved it to the navItem level
![image](https://user-images.githubusercontent.com/17003240/153571753-91d12c57-699e-4e24-bc78-cbc53a45a2ce.png)

Reply:
Thanks for moving the badge, @stratoula. Out of curiosity, is it nested inside the button itself? It looks like it's outside the button. If it's possible to include the badge inside the button, I think it may end up looking better here and also benefit by being clickable. Thoughts?

Reply:
Let me check it Michael